### PR TITLE
refactor: remove alternative tile matrix sets

### PR DESCRIPTION
**Pull request by blacha**
Fixes: #
_Add new line for each issue that was fixed_

### CODE REVIEW

File: packages/shared/src/proj/projection.tile.matrix.set.ts [hunk 1]
```diff
@@ -59,11 +53,8 @@ export class ProjectionTileMatrixSet {
      * @param epsgCode
```
Comment (Wentao-Kuang) [inline packages/shared/src/proj/projection.tile.matrix.set.ts:53]: Do we still need this?
`static altMap = new Map<EpsgCode, Map<string, ProjectionTileMatrixSet>>();`

File: packages/lambda-tiler/src/tiler.ts [hunk 2]
```diff
@@ -1,10 +1,9 @@
 import { Epsg, TileMatrixSet } from '@basemaps/geo';
 import { GoogleTms } from '@basemaps/geo/build/tms/google';
 import { Nztm2000Tms } from '@basemaps/geo/build/tms/nztm2000';
-import { Nztm2000AgolTms } from '@basemaps/shared/build/alternative.tms/nztm2000.agol';
 import { Tiler } from '@basemaps/tiler/build/tiler';
 
-export const DefaultTilers = [new Tiler(GoogleTms), new Tiler(Nztm2000Tms), new Tiler(Nztm2000AgolTms)];
+export const DefaultTilers = [new Tiler(GoogleTms), new Tiler(Nztm2000Tms)];
```
Comment (Wentao-Kuang) [inline packages/lambda-tiler/src/tiler.ts:6]: ` get(epsg: Epsg, alt?: string): Tiler | undefined {
        return Tilers.map.get(TileMatrixSet.getId(epsg, alt));
    },
`
This Tile.get and TileMatrixSet.getId can remove the alt parameter as well.

File: packages/shared/src/proj/projection.tile.matrix.set.ts (hunk 1 above)
Comment (blacha) [reply packages/shared/src/proj/projection.tile.matrix.set.ts:53]: You are right we should delete this, however it gets deleted when I remove `ProjectionTileMatrixSet` in the next PR :)

**Review by Wentao-Kuang — APPROVED** [review]

File: packages/lambda-tiler/src/tiler.ts (hunk 2 above)
Comment (blacha) [reply packages/lambda-tiler/src/tiler.ts:6]: Tile.get also gets removed in the next PR I think

**Pull request merged**